### ISSUE TITLE
Express 4.0

### ISSUE THREAD

**Issue by Kinnardian** [passport/www.passportjs.org#33]
I'd like to help updating the documentation for express 4.0 but it's a whammy and I'd also like to know if there are already any plans in place for this.

**Comment by zwhitchcox** [comment]
Agreed! It still says to use ```express.session()``` and ```express.bodyParser()``` and ```express.cookieParser```. 

[This is in the "Configure" section under "Middleware"](http://passportjs.org/docs/configure)

**Comment by Kinnardian** [comment]
Who's in charge around here?

**Comment by jaredhanson** [comment]
I am.  But you can feel free to submit PRs.

**Comment by Kinnardian** [comment]
Is there a master plan in place for updating the documentation?

**Comment by jaredhanson** [comment]
The master plan is to always be incrementally improving.  PRs welcome.

**Comment by Kinnardian** [comment]
Got it.
